pkg/lambdaworks/lambdaworks_test.go

```diff
@@ -163,7 +163,19 @@ func TestFeltDiv4Error(t *testing.T) {
 	expected := lambdaworks.FeltFromUint64(45)
 
 	result := f_four.Div(f_one)
-	if result == expected {
+	if result != expected {
```

Comment:
This test fails. It should expect the result to be different on purpose. Please, revert this change.